temporal-opentracing/src/main/java/io/temporal/opentracing/internal/SpanFactory.java

```diff
@@ -59,6 +59,18 @@ public Tracer.SpanBuilder createWorkflowStartSpan(
     return createSpan(context, tracer, null, References.FOLLOWS_FROM);
   }
 
+  public Tracer.SpanBuilder createStartNexusOperationSpan(
+      Tracer tracer, String serviceName, String operationName, String workflowId, String runId) {
+    SpanCreationContext context =
+        SpanCreationContext.newBuilder()
+            .setSpanOperationType(SpanOperationType.START_NEXUS_OPERATION)
+            .setActionName(serviceName + "." + operationName)
```

Comment:
I wonder if it'd be better to use a slash for the delimiter. That's the standard for gRPC services and likely will be the same for Nexus.

Reply:
I think this may be an opportunity for Nexus spec to define what a single-string qualified operation name looks like. I think I like slash because that represents how it looks in a URL. I also think we should consider disallowing slashes (and maybe other non-url-safe characters) in operation names, but that's not that important.

Reply:
Going with `/` for now, can resist since we are still not GA and this is the open tracing interceptor